nextflow.config

```
@@ -1,2 +1,2 @@
 params.custom_config_base = "."
-includeConfig("nfcore_custom.config")
+includeConfig ("nfcore_custom.config")
```

I think the parenthesis are not needed here
```suggestion
includeConfig "nfcore_custom.config"
```